tests/SchemaClassGeneratorTest.php

```diff
@@ -784,6 +784,13 @@ public function testGenerateQueryObjectWithObjectFields()
             static::getExpectedFilesDir() . "/query_objects/$objectName.php",
             static::getGeneratedFilesDir() . "/$objectName.php"
         );
+        
+        // Test if the right classes are generated.
+        $this->assertFileExists(static::getGeneratedFilesDir() . "/LeftQueryObject.php");
+        $this->assertFileExists(static::getGeneratedFilesDir() . "/MultipleObjectSelectorsLeftObjectsArgumentsObject.php");
+
+        $this->assertFileExists(static::getGeneratedFilesDir() . "/MultipleObjectSelectorsRightQueryObject.php");
```

Comment:
I'm wondering if you should add a comment explaining why we're doing this, the type name and field name test scenario I mean. It can be hard to understand why this is happening. Maybe a link to the pull request would do, you explained that pretty well there.

Let me know what you think.

Reply:
I will add test comments to explain what the test does.